source/_integrations/zwave_js.markdown

```diff
@@ -90,12 +90,39 @@ Use this My button:
 1. Wait for the Z-Wave JS add-on to start up.
 1. Once the installation is complete, the **Device info** of the Z-Wave controller is shown.
    * You successfully installed the Z-Wave integration and the Z-Wave JS add-on.
-   * You can now add devices to the Z-Wave network.
+   * You can now [add](/integrations/zwave_js/#adding-a-new-device-to-the-z-wave-network) devices to the Z-Wave network.
 
 <p class='note'>
 While your Z-Wave mesh is permanently stored on your dongle, the additional metadata is not. When the Z-Wave integration starts up the first time, it will interview your entire Z-Wave network. Depending on the number of devices paired with the Z-Wave dongle, this can take a while. You can speed up this process by manually waking up your battery-powered devices. Most of the time, this is a button press on those devices (see their manual). It is not necessary to exclude and re-include devices from the mesh.
 </p>
 
+### Adding a new device to the Z-Wave network
+
+1. In Home Assistant, go to <a class="my" href="https://my.home-assistant.io/redirect/integrations/" target="_blank"><b>Settings</b> > <b>Devices & Services</b></a>.
+1. In the Z-Wave integration, select **Configure**.
+1. Select **Add device**.
+   * The Z-Wave controller is now in inclusion mode and will not respond to other commands.
+1. Put the device you want to add in inclusion mode. Refer to the device manual to see how this is done.
```

Comment:
At this point there are really two options:
1. continue like it is described here, this is the "classic" manual inclusion process
2. click on "Scan QR code" to switch to add a device to the Smart Start provisioning list
   ![grafik](https://user-images.githubusercontent.com/17641229/230069042-7feae8a3-5e03-440f-8d0d-24381caa0c13.png)
   I don't think that process is documented yet.